src/app/functions/server/getPremiumSubscriptionInfo.ts

```diff
@@ -32,4 +34,17 @@ function getPremiumSubscriptionUrl({
   return `${subscriptionUrl}/products/${productId}?plan=${planId}`;
 }
 
-export default getPremiumSubscriptionUrl;
+type GetSubscriptionBillingAmount = {
+  [key in SubscriptionPeriod]: number;
+};
```

Comment:
Useful shorthand:

```suggestion
type GetSubscriptionBillingAmount = Record<SubscriptionPeriod, number>;
```

(Also I'd maybe remove the `Get` from this type, since it's not a function.)